resources/zib/zib-SOAPReport.SOAPLine.xml

```diff
@@ -37,7 +37,7 @@
       <constraint>
         <key value="zib-SOAPReport-1" />
         <severity value="error" />
-        <human value="In a SOAP report an ICPC code may be assigned, but only to the S and E line." />
+        <human value="In a SOAP report an ICPC code may be assigned, but only to the S and A line." />

```

Comment:
Hoe suf ook, dit is een fout uit de zib dus die moet in ieder geval eerst gedocumenteerd en geaccepteerd worden.

Reply:
Ik heb deze fout nu gemeld in ZIB-1920.

Reply:
Prima, we kunnen het wat mij betreft doorvoeren als het zibcentrum de fout onderkent.